RELEASENOTES.md

```diff
@@ -1,9 +1,15 @@
 # MLLP Release Notes
 
+# v1.1.0 - July 7, 2016
```

Comment:
Please recommit without the release notes. It's generally up to the owning/publishing organization to manage release notes.